test/rpc/conftest.py

```diff
@@ -54,7 +54,6 @@ def fixture_invoke_info() -> dict:
     Make an invoke transaction on devnet and return invoke info dict
     """
     invoke_tx = json.loads(INVOKE_CONTENT)
-    invoke_tx["calldata"] = ["0"]
```

Comment:
Why aren't other tests affected by this?

Reply:
the invoke test was not that much in the first place i think,

since all tests are currently passed

Reply:
Still don't understand, why was the setting of the calldata needed before this PR, but not anymore?